src/feature/articles/getSingleArticle/__tests__/GetSingleArticleComponent.test.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ViewSingleArticle } from '../GetSingleArticleComponent';
+import LikeDislikeArticle from '../../likeDislikeArticles/LikeDislikeComponent';
+import LocalStorage from '../../../../__mocks__/localStorage';
+
+let storage;
```

Comment:
Can you remove unused imports and declarations from line 5,6 and 8

Reply:
Done. Thanks.